scss/underdog/components/_dropdown.scss

```diff
@@ -17,19 +17,37 @@
   width: $dropdown-menu-width;
   z-index: layer(dropdown-menu);
 
+
+  // Fake a border-ed triangle by positioning a smaller triangle with a lighter
```

Comment:
Yup. I had to get that pixel value *just right* so that the smaller triangle would cover the larger triangle perfectly in all browsers. So annoying.